python/GafferTest/FileSequencePathFilterTest.py

```diff
@@ -297,6 +291,8 @@ def setUp( self ) :
 
 		GafferTest.TestCase.setUp( self )
 
+		self.__dir = self.temporaryDirectory().replace( "\\", "/" )
```

Comment:
In my cherry-pick PR, I've omitted this so we can consider as_posix() etc. But I have included the FileSequencePathFilter fix itself, and tidied up some other stuff in this test.